source/Calamari/TerraformCliExecutor.cs

```diff
@@ -193,7 +193,7 @@ void InitializePlugins()
             var allowPluginDownloads = variables.GetFlag(TerraformSpecialVariables.Action.Terraform.AllowPluginDownloads, true);
             string initCommand = $"init -no-color";
 
-            if (version.IsLessThan("0.15.0"))
+            if (version?.IsLessThan("0.15.0") == true)
```

Comment:
Default to bundled version, which is less than `0.15.0`, but perhaps it should be the opposite? (Assume latest)